sections/Titans stronghold.tex

```diff
@@ -0,0 +1,174 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Cooperative scenario}{Titan's stronghold}{\images/Stronghold.png}
+
+\begin{multicols*}{2}
+
+\textbf{Author:} Invoceusse
+
+\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1219333721019256943}{Archon Studio Discord}
+
+\subsection*{\MakeUppercase{Scenario length}}
+
+This scenario plays out over 16 rounds. (15 in impossible)
+
+\subsection*{\MakeUppercase{Player setup}}
+
+\textbf{Player count:} 1 - 6
+
+\textbf{Starting Resources:} 30 \svg{gold}, 8 \svg{building_materials}, 2 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 1 \svg{valuables}
+
+\textbf{Starting Units:} 
+    A Pack of \svg{bronze} units of your choice.
+    A Few \svg{bronze} units of your choice.
+
+\textbf{Town Buildings:} nothing
+
+\subsection*{\MakeUppercase{Map setup}}
+
+Take the following Map tiles and set them up as shown in the scenario map layout: (P for number of player)
+
+\textbf{(1*P) Starting Map tile (I)}
+\begin{itemize}
+    \item Starting Tiles of your chosen factions.
+    \item Ignore all of their yellow line.
+\end{itemize}
+
+\textbf{(2*P) × Far Map tile (II-III)}
+
+\textbf{(2*P) × Near Map tile (IV-V)}
+
+\textbf{(1*P) + 1 × Center Map tile (VI-VII)}
+
+If you don't have enough VI-VII tile you can use another for tile near to tile I.
+The center of this tile is the Titans' Stronghold.
+
+\columnbreak
+
+\subsection*{\MakeUppercase{Win conditions}}
+
+\textbf{Win:}
+\begin{itemize}
+	\item Kill all unit in the Titans' Stronghold (the center of tile VI-VII next to the tiles I)
+\end{itemize}
+
+\subsection*{\MakeUppercase{Timed events}}
+
+\textbf{4$^{th}$, 8$^{th}$ and 12$^{th}$ Round:}
+\begin{itemize}
+	\item Remove black cubes from windmills, water wheels and mystical gardens.
+\end{itemize}
+
+\subsection*{\MakeUppercase{The story}}
+
+TODO \end{multicols*}
+
+\newpage
+
+\subsection*{\MakeUppercase{Additional rules}}
+
+\begin{itemize}
+	\item Remember : the center of tile VI-VII next to all I, is the Titans' Stronghold.
+	\item nobody can enter the titans' Stronghold before all other VII battle are flagged by any player.
```

Comment:
> all other VII battle
What does this mean? Should it be "all other VII fields" or something like that?

```suggestion
	\item Nobody can enter the Titans' Stronghold before all other VII battle are flagged by any player.
```

Reply:
Yes, this can be "all other VII fields"